runway/util.py

```diff
@@ -79,6 +79,25 @@ def merge_dicts(dict1, dict2, deep_merge=True):
     #     return {**dict1, **dict2}
 
 
+#
+# a YAML file can be parsed into a dict with values of None!
+#
+# for example a file containing just 'foo:' will be parsed as `{'foo': None}`
+#
+# thus we can't rely on `dict.get(key, default)` as it will return `None`
+#  instead of default in those cases :-(
+#
+# not that value-less nodes are something we should encourage, but at least
+#  using this function saves us from None errors
```

Comment:
I would rather not do this for now -- can we revisit with your schematics work later?

I'd rather catch `foo: ` shenanigans as an early config/model check (ideally with a way to differentiate that from `foo: !!null`)

Reply:
Good point. And it's only used in two places.  So for now either of them being 'empty' will trigger a crash.  Later we'll explicitly reject them as mis-configured.

Removing this function.